src/dashboard/src/pages/Submission/DataJob.tsx

```diff
@@ -8,10 +8,13 @@ import {
   CardActions,
   Grid,
   Container,
+  Tooltip,
   TextField,
   Button,
   Divider, Dialog, DialogTitle, DialogContent, DialogContentText, DialogActions
 } from '@material-ui/core'
+import {Link as AmlLink}  from '@material-ui/core'
```

Comment:
Why rename `Link` here?

Reply:
Same name with component in Row 24.
import { Link } from 'react-router-dom'

Reply:
I see, there is a best practice for you to follow

https://github.com/microsoft/DLWorkspace/blob/master/src/dashboard/src/pages/Jobs/JobsTable/columns/useNameId.tsx#L7-L8
https://github.com/microsoft/DLWorkspace/blob/master/src/dashboard/src/pages/Jobs/JobsTable/columns/useNameId.tsx#L43-L45